utils/fnIt.ts

```diff
@@ -0,0 +1,13 @@
+import { spy } from "ts-mockito";
+
```

Comment:
This utils folder is at the top level. I think you should put it in the /test/ folder